lib/etl/load/create_new_group_records.rb

```diff
@@ -0,0 +1,32 @@
+class CreateNewGroupRecords
+  def write(attributes_list)
+    @attributes_list = attributes_list
+
+    attributes_list.each do |attributes|
+      puts "#{attributes["name"]}"
+      
+      if existing_record attributes
+        ap "Name already in database: #{name attributes}" # rubocop:disable Rails/Output
+      else
+        record = Group.create!(attributes)
+
+        ap "Name: #{record.name}" # rubocop:disable Rails/Output
+        ap 'Metadata: ' # rubocop:disable Rails/Output
+        ap record.metadata # rubocop:disable Rails/Output
+        ap record.long_description # rubocop:disable Rails/Output
+      end
+    end
+  end
+
+  private
+
+  attr_reader :attributes_list
+
+  def existing_record(attributes)
+    Group.find_by(name: name(attributes))
```

Comment:
I didn't want there to be duplicates when running an ingestion script multiple times.  The group name may not be the best key, just as title for resources isn't the best.  Some kind of other unique ID may be the way to go...but that's a larger discussion.